feature-libs/product-configurator/rulebased/components/form/configurator-form.component.ts

```diff
@@ -76,6 +78,9 @@ export class ConfiguratorFormComponent implements OnInit {
               }
             });
         }
+        if (routingData.expMode) {
+          this.configExpertModeService.setExpMode(routingData.expMode);
+        }
```

Comment:
seem like that there was no unit test added for this condition

Reply:
Done!